lightkurve/lightcurvefile.py

```diff
@@ -139,10 +143,16 @@ def from_archive(target, cadence='long', quarter=None, month=None,
         -------
         lcf : KeplerLightCurveFile object or list of KeplerLightCurveFile objects
         """
-        path = download_kepler_products(
-            target=target, filetype='Lightcurve', cadence=cadence,
-            quarter=quarter, campaign=campaign, month=month,
-            radius=radius, targetlimit=targetlimit)
+        # Be tolerant if a direct path or url is passed to this function by accident
+        if os.path.exists(str(target)) or str(target).startswith('http'):
```

Comment:
Could also check if it ends with ".fits"? Here if I pass it a path that happens to not exist we're still going to get that error. Checking if it ends in "fits" will probably stop that.

Reply:
We don't need to check whether the path or url is valid as part of `from_archive`, we can rely on the standard constructor (i.e. `LightCurveFile.__init__()`) to give feedback on a file or url being invalid.